symplyphysics/laws/kinematic/constant_accelerated_movement_is_parabolic.py

```diff
@@ -7,47 +7,44 @@
 
 # Description
 ## Accelerated movement is the kind of movement when object has constant acceleration (e.g with the constant force applied to object).
-## For example, free fall is accelerated movement. The acceleration in each moment of time is g.
-## According to velocity and acceleration definitions, velocity is movement derivative, acceleration is velocity derivative.
-## The law is: S = V0*t + a * t**2/2, where
-## S is distance in the moment of time t
-## V0 is the velocity in the moment of time 0 (initial velocity)
-## a is acceleration
+## The movement is S = V0*t + a * t**2/2, where
```

Comment:
`constant_accelerated_movement` for file name does not sound right. Maybe `constant_acceleration_movement` is better?